src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.*;
+
+/**
+ * Association class
+ * 
+ *
+ */
+
+public class Tagging {
+	
+	private boolean isAdded = false;
+	private Tag tag;
+	private Name name;
```

Comment:
assuming the application allows multiple people with same name, how would this hamper the tagging list? think!
what would be a better way to handle the association?